### ISSUE TITLE
cnSegmentation for EPICv2 data

### ISSUE THREAD

**Issue by SteffanChristiansen** [zwdzwd/sesame#143]
It seems like there are no default reference data for the EPICv2 platform for SeSAMe v. 1.20.0. 

`sdf <- openSesame(searchIDATprefixes(data_dir),
                  manifest = addr,
                  func = NULL,
                  prep = "",
                  platform = "EPICv2")`
`segs <- cnSegmentation(sdf[[1]])`

Error in cnSegmentation(sdf[[1]]) : 
  Please provide sdfs.normal=. No default for EPICv2

Do you plan any updates to include that in the near future or do you know a suitable reference that you can recommend for the task of identifying copy number segments in a tumor cell line?

**Comment by zwdzwd** [comment]
Yes, the reference has been added to v.1.21.x. See also the following https://github.com/zwdzwd/sesame/issues/129 for previous versions.

**Comment by SteffanChristiansen** [comment]
Thanks for the suggestion. I will let you know when I have tested the solution. Unfortunately, I am facing the same issue as described in #145 and in #127 described by @bethan-mallabar-rimmer.

**Comment by zwdzwd** [comment]
Have you tried upgrading sesameData, or do you have problem mainly with caching?

**Comment by SteffanChristiansen** [comment]
Yes, I removed the cache dir, upgraded to sesameData and SeSAMe v 1.21 (or above) cached again. Still, I was not able to cache all of the data as described in #145 . 

Returned to version 1.20 afterwards as I was no longer able to run several of the functions.

**Comment by seq101** [comment]
> Yes, I removed the cache dir, upgraded to sesameData and SeSAMe v 1.21 (or above) cached again. Still, I was not able to cache all of the data as described in #145 .
> 
> Returned to version 1.20 afterwards as I was no longer able to run several of the functions.

@SteffanChristiansen @zwdzwd  @HenrikBengtsson  @karltk  I am having this same problem. When I upgrade Sesame to developer version to use defaults for Epic V2, the cache stops to work making many other functions to fail.

**Comment by arualemsti** [comment]
> > Yes, I removed the cache dir, upgraded to sesameData and SeSAMe v 1.21 (or above) cached again. Still, I was not able to cache all of the data as described in #145 .
> > Returned to version 1.20 afterwards as I was no longer able to run several of the functions.
> 
> @SteffanChristiansen @zwdzwd @HenrikBengtsson @karltk I am having this same problem. When I upgrade Sesame to developer version to use defaults for Epic V2, the cache stops to work making many other functions to fail.

I'm also having a DataCache issue. Just wondering if anyone has found a solution?
